### PR TITLE
Tests: Improve Percy Font Loading Flakiness

### LINKED ISSUE #6317: Percy Font Loading Flakiness
Nothing new here really, ticket for tracking purposes. 

Seems that snapshots are sometimes taken at a marginally different size or speed and reporting 'changes' when nothing actually changed. 

For example, I had to approve percy changes in a recent documentation update I made here: https://percy.io/google/web-stories-wp/builds/8765894/changed/497717303?browser=chrome&browser_ids=14%2C16&subcategories=approved&viewLayout=side-by-side&viewMode=new&width=1600&widths=1600  where the only diff was a markdown file.

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Add array of font weights and promise.all. 

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [ ] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] I have tested this code to the best of my abilities
- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6317

### CODE REVIEW

**Comment by swissspidy** [comment]
Also need to actually load Roboto 700  in the `client_with_context.html`.

In `Story_Post_Type` we use this:

```
https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap
```

And this in the `Dashboard`:

```
https://fonts.googleapis.com/css?family=Google+Sans|Google+Sans:b|Google+Sans:500
```

**Comment by swissspidy** [comment]
The percy flakiness in the e2e tests is also  very interesting: https://percy.io/google/web-stories-wp-e2e/builds/8804891/changed/499793964?browser=chrome&browser_ids=14%2C16&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1600&widths=1600

I think it's because of this:

https://github.com/google/web-stories-wp/blob/ef1f1c45173422b937f2df566338a16207c81db2/assets/src/edit-story/elements/media/display.js#L41

https://github.com/google/web-stories-wp/blob/ef1f1c45173422b937f2df566338a16207c81db2/assets/src/edit-story/elements/media/imageDisplay.js#L74-L95

If the image hasn't been fully loaded, it has a grey background.

Perhaps we could just add some percy-specific CSS to always remove that background color?

**Comment by spacedmonkey** [comment]
> Also need to actually load Roboto 700 in the client_with_context.html.

🤦 Of course I do!! Fixed

File: packages/e2e-tests/src/specs/dashboard/templates/useTemplate.js [hunk 1]
```diff
@@ -20,6 +20,8 @@
 import { percySnapshot } from '@percy/puppeteer';
 import { visitDashboard } from '@web-stories-wp/e2e-test-utils';
 
+const percyCSS = `.dashboard-media-element { background-color: none; }`;
```
Comment (swissspidy) [inline packages/e2e-tests/src/specs/dashboard/templates/useTemplate.js]: This flakiness is not isolated to this test, but could occur anywhere media elements are displayed. So it's probably better to move this to `percy.config.yml`.

File: assets/src/edit-story/elements/media/display.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ import { useUnits } from '../../units';
 import { getMediaWithScaleCss } from './util';
 import getMediaSizePositionProps from './getMediaSizePositionProps';
 
-const Element = styled.div`
+const Element = styled.div.attrs({ className: 'dashboard-media-element' })`
```
Comment (swissspidy) [inline assets/src/edit-story/elements/media/display.js]: Why `dashboard`? We're in the editor :-)

Perhaps `story-media-display-element`?

File: assets/src/karma-fixture/client_with_context.html [hunk 3]
```diff
@@ -12,11 +12,11 @@
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
   <link
-    href="https://fonts.googleapis.com/css?family=Google+Sans|Google+Sans:b&display=swap"
+    href="https://fonts.googleapis.com/css?family=Google+Sans|Google+Sans:b|Google+Sans:500"
```
Comment (swissspidy) [inline assets/src/karma-fixture/client_with_context.html]: Removed the `display=swap` on purpose? I think we should keep it.

**Review by swissspidy — APPROVED** [review]
One nit but otherwise LGTM 👍

File: assets/src/karma-fixture/client_with_context.html (hunk 3 above)
Comment (spacedmonkey) [reply assets/src/karma-fixture/client_with_context.html]: It wasn't part loading like this in dashboard. So I making load the same way in both places. 
https://github.com/google/web-stories-wp/pull/6324/commits/96c02aff82c07aae37a4676af04566e9271a0ee1

Comment (swissspidy) [reply assets/src/karma-fixture/client_with_context.html]: even better!

**Pull request merged**